{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -43,9 +43,15 @@ basepython =
     py27: {env:TOXPYTHON:python2.7}
     py34: {env:TOXPYTHON:python3.4}
     py35: {env:TOXPYTHON:python3.5}
-    {% if cookiecutter.sphinx_docs == "yes" %}{py36,docs,spell}{% else %}py36{% endif %}: {env:TOXPYTHON:python3.6}
+    py36: {env:TOXPYTHON:python3.6}
+{%- if cookiecutter.sphinx_docs == "yes" %}
+    {docs,spell}: {env:TOXPYTHON:python{{ cookiecutter.python_version_to_use_for_checks }}}
+{%- endif %}
     py37: {env:TOXPYTHON:python3.7}
     {bootstrap,clean,check,report{%- if cookiecutter.codecov == 'yes' %},codecov{% endif %}{% if cookiecutter.coveralls == 'yes' %},coveralls{% if cookiecutter.c_extension_support != "no" %},extension-coveralls{% endif %}{% endif %}}: {env:TOXPYTHON:python3}
+envdir =
+    {toxworkdir}/{envname}
+    {docs,spell,check}: {toxworkdir}/py{{ cookiecutter.python_version_to_use_for_checks | replace('.', '') }}
```

Comment:
I think this will make two envs with mismatching deps use the same envdir, thus make the virtualenvs rebuild every time you run something.

Reply:
I don't...think...that it's doing that. At least, the first time I run tox, it says:
```
check create: /tmp/python-nameless/.tox/check
/tmp/python-nameless/.tox/py36
docs create: /tmp/python-nameless/.tox/docs
/tmp/python-nameless/.tox/py36
```
The second time I run tox, it skips straight to `check isntalled...`. It still says "check installed", but it doesn't seem to take long enough to be actually installing anything, so I think it's actually re-using the previously installed stuff. I don't know how to be absolutely sure that it isn't rebuilding every time.

When I look at `ls -l .tox/py36/lib/python3.6/site-packages/` before and after running tox, running tox doesn't reset the timestamps. But I'm not sure if that's where I should be looking.

Reply:
I think weird stuff will happen as soon as you start adding deps into either env configuration.

Reply:
> I think weird stuff will happen as soon as you start adding deps into either env configuration.

It'll recreate the environment if any of the deps change, if that's what you mean. E.g. I just added sphinx to the deps of `check`, and even though sphinx was obviously already installed (for `docs`), it recreated the env.
```
$ tox -e check
check recreate: /tmp/python-nameless/.tox/check
/tmp/python-nameless/.tox/py36
check installdeps: docutils, check-manifest, flake8, readme-renderer, pygments, isort, sphinx
```
That's weird, true, but is it harmful?

Reply:
Well it is annoying to have envs rebuild everytime i run `tox -e check,py36`.